import-export-cli/utils/constants.go

```diff
@@ -105,3 +105,5 @@ const ApplicationId = "applicationId"
 const ApiId = "apiId"
 const DefaultCliApp = "default-apictl-app"
 const DefaultTokenType = "JWT"
+
+var ValidInitialStates = []string{"PUBLISHED"}
```

Comment:
If someone put initial-state as "CREATED". It would get invalidated if we put only "PUBLISHED" here.

Reply:
@pubudu538

Reply:
@renuka-fernando can we fix this? we should allow CREATED and PUBLISHED both